lib/chef-dk/command/generator_commands.rb

```diff
@@ -36,7 +36,10 @@ module SharedGeneratorOptions
       option :license,
         :short => "-I LICENSE",
         :long => "--license LICENSE",
-        :description => "all_rights, apache2, mit, gplv2, gplv3 - defaults to all_rights",
+        :description => "all_rights, apachev2, mit, gplv2, gplv3 - defaults to all_rights",
+        :proc => Proc.new { |l|
+          l.match(/apache.?2/) ? 'apache2' : l
+        },
```

Comment:
kind of feels weird to force the input to 'apache2' here and then also have the case statement around 'apachev2' which should never happen.  also kind of weird that 'apachev2' seems to be canonical in the help but we convert that internally.  also kind of weird that 'apachewtf2' will get converted into 'apache2'.  should probably change this to proc to convert 'apache2' to 'apachev2' and then drop the 'apache2' from the case statement below...